### PR TITLE
Set max string length for ruff

### PR DESCRIPTION
**Pull request by gsheni**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -225,7 +225,11 @@ lines-between-types = 0
 [tool.ruff.lint.per-file-ignores]
 "__init__.py" = ["F401", "E402", "F403", "F405", "E501", "I001"]
 "errors.py" = ["D105"]
-"tests/**.py" = ["D"]
+"tests/**.py" = ["D", "W505"]
```
Comment (amontanez24) [inline pyproject.toml:228]: why are we ignoring for the tests?

Comment (gsheni) [reply pyproject.toml:228]: - When we enabled the length checking, I got error from the test directory.
- I figured that checking docstring length check (`W505`) was not important for the tests directory.

Comment (amontanez24) [reply pyproject.toml:228]: I think we should apply it everywhere. The main use case as I see it is to make it easier to read and review, so I think tests would still have that problem

Comment (gsheni) [reply pyproject.toml]: Fixed:
https://github.com/sdv-dev/SDV/pull/2095/commits/02b804a5f2a397e4ca7179a5dc58f56980a193d6

**Review by amontanez24 — APPROVED** [review]

**Review by pvk-developer — APPROVED** [review]

File: tests/unit/sequential/test_par.py [hunk 2]
```diff
@@ -469,8 +469,8 @@ def test__fit_context_model_with_context_columns(self, gaussian_copula_mock):
 
     @patch('sdv.sequential.par.PARSynthesizer.get_transformers')
     def test_auto_assign_transformers_without_enforce_min_max(self, mock_get_transfomers):
-        """Test to see if auto_assign_transformers does not add enforce_min_max_values if the transformer
-        does not contain it already
+        """Test to see if auto_assign_transformers does not add enforce_min_max_values
+        if the transformer does not contain it already
```
Comment (pvk-developer) [inline tests/unit/sequential/test_par.py:473]: I think that this should be failing 🤔

File: pyproject.toml [hunk 3]
```diff
@@ -229,3 +229,7 @@ lines-between-types = 0
 
 [tool.ruff.lint.pydocstyle]
 convention = "google"
+
+[tool.ruff.lint.pycodestyle]
+max-doc-length = 100
+max-line-length = 100
```
Comment (R-Palazzo) [inline pyproject.toml:235]: Should it be 100 or 99? On Enterprise it's 99

**Pull request merged**